### ISSUE TITLE
Clicking spyglass icon in raw_id_fields widget does not popup a new window

### ISSUE THREAD

**Issue by odedfos** [darklow/django-suit#364]
related to: https://github.com/darklow/django-suit/issues/360

This happens when updating to django 1.8.

When clicking the magnifying glass icon, the browser is redirected to the relevant admin list view, which is wrong.
The Correct behavior is for the list view to appear in a popup, and selecting an item from it should set the value in the widget.

**Comment by darklow** [comment]
Yesterday i approved pull request that should solve this issue.
Please try latest Django suit develop version and see if it fixes the problem.
```
pip install https://github.com/darklow/django-suit/tarball/develop
```

**Comment by odedfos** [comment]
It works :)

**Comment by vladlep** [comment]
nice, this solves the issue:). Is there also  a new release that i could update to? I would not to pull from the develop branch as it could break things in the future. Thanks.

**Comment by darklow** [comment]
yes as this was while ago, this fix should be in latest version too:
`pip install django-suit==0.2.14`

**Comment by vladlep** [comment]
thanks. I also noticed now that the search icon does not appear if you have a ForeignKey to a ProxyModel in django 1.8.  I am not sure if this was supported in general? 
I am now trying to discover the best design on my database and I am trying out different options with inheritance.

**Comment by marco-santamaria** [comment]
I tried with version 0.2.14 and 0.2.15 with no success
Also

    pip install https://github.com/darklow/django-suit/tarball/develop

didn't work

**Comment by SalahAdDin** [comment]
This is big problem!

**Comment by tiagovaz** [comment]
Hi there, I'm using django 1.9.5 with django-suit 0.2.18. No popus :( The workaround from #360 doesn't work neither.

**Comment by tiagovaz** [comment]
I've just deleted my last comment, I was using a js blocker... duh

django 1.9.5 + django-suite 0.2.18 works perfectly, thanks!

**Comment by LeMeteore** [comment]
Hey guys o/
Just to mention that I'm experiencing the same issue. I'm using `django==1.8.18` and `django-suit==0.2.18`.
